### PR TITLE
kv/client: fix time is not updated when receiving committed event

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Fix a bug in kv client v2 that committed events are not counted in event ts manager

### What is changed and how it works?

- Always send ts update notification to event ts manager when region receives any event and it is not initialized.
- Remove a failpoint in test case, and we can update global variable directly.

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

**Review by leoppro — APPROVED** [review]

File: cdc/kv/client_test.go [hunk 1]
```diff
@@ -2803,11 +2803,11 @@ func (s *etcdSuite) testKVClientForceReconnect(c *check.C) {
 
 	err = failpoint.Enable("github.com/pingcap/ticdc/cdc/kv/kvClientResolveLockInterval", "return(1)")
 	c.Assert(err, check.IsNil)
-	err = failpoint.Enable("github.com/pingcap/ticdc/cdc/kv/kvClientReconnectInterval", "return(3)")
-	c.Assert(err, check.IsNil)
+	originalReconnectInterval := reconnectInterval
+	reconnectInterval = 3 * time.Second
```
Comment (liuzix) [inline cdc/kv/client_test.go:2807]: Is it possible for the race detector to complain about this?

**Comment by leoppro** [comment]
any command to cancel LGTM? /unlgtm

**Comment by leoppro** [comment]
> any command to cancel LGTM? /unlgtm

fine, I have to review this PR ASAP

File: cdc/kv/client_test.go (hunk 1 above)
Comment (amyangfei) [reply cdc/kv/client_test.go:2807]: No, the test cases run in sequence. (The original code also changes the global value, but triggered by the failpoint)

**Comment by amyangfei** [comment]
> any command to cancel LGTM? /unlgtm

@leoppro try `/lgtm cancel`

**Pull request merged**